### ISSUE TITLE
Riddle Transfer 1 menu does not work.

### ISSUE THREAD

**Issue by TheTechRobo** [lightspark/lightspark#657]
### What is the purpose of your issue?
- [X] Site Support Request: Lightspark does not work correctly on a specific website. (I think)
- [ ] Bug: Lightspark has the same problem across multiple websites or SWF files.
- [ ] Feature request: a request for new features or support for new platforms, *not sites or SWF files*.
- [ ] Question, Comment, or Suggestion
- [ ] If none of the above apply, check this box.

#### Provide the complete URL of the page with the issue here:

you might find it on Newgrounds but didn't test - my browser doesn't have it installed since i'm using chromeos

#### If it is possible to download the file and run it with the command line, please provide a link to the file here:

http://ahkong.net/riddle-transfer-download/

#### Provide the complete set of instructions required to reproduce the problem:

1. Load the SWF file.
2. see lag (maybe)
3. try to click main menu button
4. See that the button doesn't take you to the menu (or if it does, it was too slow for me to notice with the lag)

#### What kind of failure is your issue?

- [ ] **Fault**: You receive a black window instead of the flash content with a white text such as ``We're sorry, Lightspark encountered an as yet unsupported Flash file``
- [ ] **Crash**: The window disappears or your browser crashes with a message such as ``A tab has crashed``
- [X] **Broken**: Lightspark shows something, but most of the file is not usable
- [ ] **Glitch**: Lightspark shows something and most of the file is usable, but things are missing or wrong compared to official Flash.
- [X] **Performance**: Lightspark runs slow or uses lots of memory trying to play the file

#### Provide the Lightspark version

Output of ``lightspark -v``:

`INFO: Lightspark version 0.8.4-880-g8a9ea814 Copyright 2009-2013 Alessandro Pignotti and others`

I built from source about an hour ago.

#### What operating system are you are running on?

- [ ] Windows 7, 8 or 10
- [ ] Windows XP, Vista or older - I didn't know that Win2000 worked on LightSpark. Or actually the other way round.
- [X] Linux: Debian Buster (running in Crostini on a chromebook from sometime recent (probably anywhere from 2018 to mid 2020))
- [ ] Mac OS X: *<specify version here, such as 10.8.1>*
- [ ] Other: *<specify OS and version here>*

#### How are you are running Lightspark?

- [ ] Browser plugin for Chrome or Chromium: *<specify Chromium version here>*
- [ ] Browser plugin for Waterfox: *<specify Waterfox version here>*
- [ ] Browser plugin for other browser: *<specify browser and version here>*
- [X] Lightspark executable on the command line

#### Which of these actions you have taken to help debug the problem?

- [X] I attached a screen shot of the problem to this issue
- [ ] I have a backtrace listing and pasted it in the bottom section of this issue (*hint: hitting Ctrl-C after a crash will copy it to the clipboard*)
- [X] I have reproduced the problem using the Lightspark command line executable
- [X] I ran `lightspark -l 4` on the command line and am attaching logs to this issue (*hint: if they are too big, zip them*)

### Description of your issue

starting screen is laggy and you cannot continue past it - main menu button does not work.

here is a video of the problem (the video is a bit laggy but most of that lag is from lightspark, and it's not that laggy in adobe flash standalone (and it actually works)) In the vidoe i am trying to click the main menu button but all it does is animate - no main menu. Sorry for the external link, I didn't want to convert between formats.

https://www.youtube.com/watch?v=qRG6uk0vO3Q

Here is the log that I got. 33mb unzipped lol.

[rt1.zip](https://github.com/lightspark/lightspark/files/5940225/rt1.zip)

**Comment by eiglow** [comment]
I just did some testing and found that it crashes since a2f0b7baa4ae41d222ceb8cd7231b5ec674eb1c1, so I filed a bug for that (#700).

Before that commit though, I tested on 798f83dc5b030907490f49fede84e223d0482539 and the initial "MAIN MENU" button, as well as the later menu buttons both work. Although you might need to click on certain parts of the button, or try multiple times, but it mostly works fine. Not perfect though.
